WordPress/src/main/java/org/wordpress/android/ui/prefs/ExperimentalFeaturesActivity.kt

```diff
@@ -0,0 +1,184 @@
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Switch
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBar
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import dagger.hilt.android.AndroidEntryPoint
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.flow.update
+import org.wordpress.android.R
+import org.wordpress.android.ui.compose.theme.AppThemeM3
+import org.wordpress.android.ui.compose.unit.Margin
+import org.wordpress.android.util.extensions.setContent
+
+val experimentalFeatures = listOf(
+    Feature(key = "experimental_block_editor", label = "Experimental block editor"),
+    Feature(key = "experimental_block_editor_theme_styles", label = "Experimental block editor styles")
+)
+
+data class Feature(
+    val enabled: Boolean = false,
+    val key: String,
+    val label: String
+)
+
+class FeatureViewModel : ViewModel() {
+    private val _switchStates = MutableStateFlow<Map<String, Feature>>(emptyMap())
+    val switchStates: StateFlow<Map<String, Feature>> = _switchStates.asStateFlow()
+
+    init {
+        val initialStates = experimentalFeatures.associate { item ->
+            item.key to Feature(AppPrefs.getManualFeatureConfig(item.key), item.key, item.label)
+        }
+        _switchStates.value = initialStates
+    }
+
+    fun toggleFeature(key: String, label: String, enabled: Boolean) {
+        _switchStates.update { currentStates ->
+            currentStates.toMutableMap().apply {
+                this[key] = Feature(enabled, key, label)
+                AppPrefs.setManualFeatureConfig(enabled, key)
+            }
+        }
+    }
+}
+
+@AndroidEntryPoint
+class ExperimentalFeaturesActivity : AppCompatActivity() {
+    private val viewModel: FeatureViewModel by viewModels()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        setContent {
+            AppThemeM3 {
+                val features by viewModel.switchStates.collectAsStateWithLifecycle()
+
+                ExperimentalFeaturesScreen(
+                    features = features,
+                    onFeatureToggled = viewModel::toggleFeature,
+                    onNavigateBack = onBackPressedDispatcher::onBackPressed
+                )
+            }
+        }
+    }
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun ExperimentalFeaturesScreen(
+    features: Map<String, Feature>,
+    modifier: Modifier = Modifier,
```

Comment:
We don't pass a `Modifier` to this function, so I believe this parameter is unnecessary. We can likewise remove the parameter from `FeatureToggle`.